src/inmanta/parser/plyInmantaLex.py

```diff
@@ -67,7 +67,7 @@
 
 def t_FSTRING(t: lex.LexToken) -> lex.LexToken:  # noqa: N802
     r"f(\"([^\\\"\n]|\\.)*\")|f(\'([^\\\'\n]|\\.)*\')"
-    t.value = t.value[2:-1]
+    t.value = safe_decode(token=t, warning_message="Invalid escape sequence in f-string.", start=2, end=-1)
```

Comment:
This fixes what I believe to be another bug namely escape characters not being displayed properly in f-strings:
Given model:
```
data = 10
std::print(f"\n{ data }")
```
output before change:
```
\n10
```

output after change:
```

10
```

Reply:
I think this is a good change. @wouterdb could you pay special attention to this?